docs/components/textarea-field/index.md

```diff
@@ -242,6 +242,47 @@ Target the error block via `data-error`.
   />
 </div>
 
+### TextareaField with character count 
+
+<div class='mb-4 w-64'>
+  <Form::Fields::Textarea
+    @label='Label'
+    @hint='With hint text'
+  >
+    <:secondary as |secondary|>
+      <secondary.CharacterCount @max={{255}} />
+    </:secondary>
+  </Form::Fields::Textarea>
+</div>
```

Comment:
Looking at [the input character count PR](https://github.com/CrowdStrike/ember-toucan-core/pull/115/files#diff-a8b8b52bc437dd62131ac31ef0ee1eb0d6bfa4455432b2b9a230522db1a184b9R243), it looks like we set the `class="w-full"` on the demos here.  We may want to do the same for consistency reasons here!

```suggestion
<div class='mb-4 w-64'>
  <Form::Fields::Textarea
    class="w-full"
    @label='Label'
    @hint='With hint text'
  >
    <:secondary as |secondary|>
      <secondary.CharacterCount @max={{255}} />
    </:secondary>
  </Form::Fields::Textarea>
</div>
```